<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Channel Watch</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resource.qrc">
    <normaloff>:/assets/images/wave.png</normaloff>:/assets/images/wave.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <property name="childrenCollapsible">
       <bool>true</bool>
      </property>
      <widget class="QTableView" name="electricalTable">
       <property name="editTriggers">
        <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
       </property>
      </widget>
      <widget class="QTableView" name="magneticTable">
       <property name="editTriggers">
        <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionCoefficient"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuRefresh_interval">
     <property name="title">
      <string>Refresh interval</string>
     </property>
     <addaction name="action1ms"/>
     <addaction name="action10ms"/>
     <addaction name="action100ms"/>
     <addaction name="action500ms"/>
     <addaction name="action1m"/>
     <addaction name="action3m"/>
     <addaction name="action5m"/>
    </widget>
    <addaction name="actionAlway_on_top"/>
    <addaction name="separator"/>
    <addaction name="menuRefresh_interval"/>
   </widget>
   <addaction name="menuTools"/>
   <addaction name="menuWindow"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCoefficient">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Coefficient</string>
   </property>
  </action>
  <action name="actionAlway_on_top">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Alway on top</string>
   </property>
  </action>
  <action name="action1ms">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1 ms</string>
   </property>
  </action>
  <action name="action10ms">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>10 ms</string>
   </property>
  </action>
  <action name="action100ms">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>100 ms</string>
   </property>
  </action>
  <action name="action500ms">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>500 ms</string>
   </property>
  </action>
  <action name="action1m">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1 m</string>
   </property>
  </action>
  <action name="action3m">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3 m</string>
   </property>
  </action>
  <action name="action5m">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>5 m</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../resource.qrc"/>
 </resources>
 <connections/>
</ui>
